crates/biome_js_analyze/tests/specs/nursery/useSemanticElements/invalid.jsx.snap

```diff
@@ -442,16 +459,19 @@ invalid.jsx:7:10 lint/nursery/useSemanticElements ━━━━━━━━━━
 ```
 invalid.jsx:8:10 lint/nursery/useSemanticElements ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
 
-  ! The element with this role can be changed to a DOM element that already this role.
+  ! The elements with the following roles can be changed to the following elements:
+    <button>
+    <input type="button">
```

Comment:
If we're able to suggest possible alternatives like in this case, why do we keep showing the loooong list at the end?

Reply:
Thank you for reviewing.
I modified suggestion messages in https://github.com/biomejs/biome/pull/2930/commits/207c4f441d0feb18b28c74d6bb7b3ce8e782efdf. 👍